### PR TITLE
Fix build scripts on Mac

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: hack/update-bazel.sh [hunk 1]
```diff
@@ -20,6 +20,15 @@ set -o pipefail
 export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+if LANG=C sed --help 2>&1 | grep -q GNU; then
```
Comment (cblecker) [inline hack/update-bazel.sh]: note to self: this snippet is in so many scripts.. should probably be ripped into hack/lib/util.sh as a function.

Comment (sttts) [reply hack/update-bazel.sh]: Indeed, found a couple of them.

**Pull request merged**